app/src/main/res/values-sw360dp/dimens.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="sp_12">12sp</dimen>
+    <dimen name="sp_14">14sp</dimen>
+    <dimen name="sp_16">16sp</dimen>
+    <dimen name="sp_18">18sp</dimen>
+    <dimen name="dp_0.5">0.5dp</dimen>
+    <dimen name="dp_3">3dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_10">10dp</dimen>
+    <dimen name="dp_24">24dp</dimen>
+    <dimen name="dp_300">300dp</dimen>
+    <dimen name="dp_50">50dp</dimen>
+    <dimen name="dp_20">20dp</dimen>
+    <dimen name="dp_15">15dp</dimen>
+    <dimen name="dp_80">80dp</dimen>
+    <dimen name="dp_35">35dp</dimen>
+    <dimen name="dp_2">2dp</dimen>
+    <dimen name="dp_45">45dp</dimen>
+    <dimen name="sp_30">30sp</dimen>
+    <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_100">100dp</dimen>
+    <dimen name="dp_40">40dp</dimen>
+    <dimen name="sp_20">20sp</dimen>
+    <dimen name="sp_17">17sp</dimen>
+    <dimen name="dp_90">90dp</dimen>
+    <dimen name="dp_30">30dp</dimen>
+    <dimen name="dp_170">170dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_170` appears to be unused
 Unused resources make applications larger and slow down builds.